src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7377,7 +7377,6 @@ components:
       description: The response object for a submitted SnapshotAccessRequest.
       required:
         - id
-        - datasetId
```

Comment:
I think `required` only has an effect on schemas used for requests, not responses. Does this setting have an effect? If not, should we remove `required` from this schema?

Reply:
I think you're right. I'll remove it